src/python/views/demo_tigergraph_circle/__init__.py

```diff
@@ -0,0 +1,774 @@
+            Graphistry Load Time (s): {float(metrics['graphistry_time']):0.2f} | 
+            Node Count: {metrics['node_cnt']} |  
+            Edge Count: {metrics['edge_cnt']} | 
+            Property Count: {metrics['prop_cnt']}  
+        </small>''', unsafe_allow_html=True)
+
+
+############################################
+#
+#   PIPELINE FLOW
+#
+############################################
+
+
+def run_all():
+    custom_css()
+
+    try:
+
+        # Render sidebar and get current settings
+        sidebar_filters = sidebar_area()
+
+        # Compute filter pipeline (with auto-caching based on filter setting inputs)
+        # Selective mark these as URL params as well
+        if sidebar_filters is None:
+            return
+       
+        filter_pipeline_result = run_filters(**sidebar_filters)
+
+        # Render main viz area based on computed filter pipeline results and sidebar settings if data is returned
+        if filter_pipeline_result['nodes_df'].size > 0:
+            main_area(filter_pipeline_result['url'],
+                      filter_pipeline_result['nodes_df'],
+                      filter_pipeline_result['edges_df'],
+                      sidebar_filters['user_id'],
+                      filter_pipeline_result['conn'])
+        else:  # render a message
+            st.write("No data matching the specfiied criteria is found")
+
+    except Exception as exn:
+        st.write('Error loading dashboard')
+        st.write(exn)
+# import asyncio
+# import graphistry
+# import os
+# import pandas as pd
+# import streamlit as st
+# from components import GraphistrySt, URLParam
+# from neptune_helper import gremlin_helper, df_helper
+# from css import all_css
+# from util import getChild
+# import time
+# import altair as alt
+# from TigerGraph_helper import tg_helper
+# import plotly.express as px
+#
+# from gremlin_python import statics
+# from gremlin_python.process.graph_traversal import __
+# from gremlin_python.process.traversal import WithOptions, T
+#
+# ############################################
+# #
+# #   DASHBOARD SETTINGS
+# #
+# ############################################
+# #  Controls how entrypoint.py picks it up
+#
+#
+# app_id = 'tigergraph_circle'
+# logger = getChild(app_id)
+# urlParams = URLParam(app_id)
+# node_id_col = 'id'
+# src_id_col = 'from_id'
+# dst_id_col = 'to_id'
+# node_label_col = 'Source_Type'
+# edge_label_col = 'Destination_Type'
+#
+# # Setup a structure to hold metrics
+# metrics = {'tigergraph_time': 0, 'graphistry_time': 0,
+#            'node_cnt': 0, 'edge_cnt': 0, 'prop_cnt': 0}
+#
+#
+# conn = tg_helper.connect_to_tigergraph()
+#
+# # Define the name of the view
+# def info():
+#     return {
+#         'id': app_id,
+#         'name': 'TigerGraph: Fraud Filter circle',
+#         'tags': ['demo', 'tigergraph_demo_circle']
+#     }
+#
+#
+# def run():
+#     run_all()
+#
+#
+# ############################################
+# #
```

Comment:
Should we remove these, esp. as this is for pedagogic purposes?

Reply:
Comments were removed in latest push